plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/behavior/OpenShiftLaunchController.java

```diff
@@ -119,8 +121,8 @@ public void launch(ILaunchConfiguration configuration, String mode,
 		
 		
 		try {
-			IDeploymentConfig dc = OpenShiftServerUtils.getDeploymentConfig(server);
-			toggleDebugging(mode, monitor, beh, server, dc);
+			IReplicationController rc = OpenShiftServerUtils.getReplicationController(server);
```

Comment:
@bdshadow good catch. I believe that #startDebugging should thus null-check the debugContext.